packages/taco/src/conditions/base/rpc.ts

```diff
@@ -12,9 +16,9 @@ export const rpcConditionSchema = z.object({
   chain: createUnionSchema(SUPPORTED_CHAIN_IDS),
   method: z.enum(['eth_getBalance', 'balanceOf']),
```

Comment:
I don't believe `balanceOf` is an RPC call; it's an ERC20 contract call (ContractCondition should be used for that). Only `eth_getBalance` should be allowed here for now. On the `nucypher` side, only `eth_getBalance` is allowed - https://github.com/nucypher/nucypher/blob/v7.0.0/nucypher/policy/conditions/evm.py#L127